setup.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+from os import path
+from setuptools import setup, find_packages
+
+
+pkg_name = "intanrhdreader"
+here = path.abspath(path.dirname(__file__))
```

Comment:
are we standardizing around `pathlib` or `os.path`?

Reply:
We haven't.  Both DataJoint Python and Elements use both.  Do you have preference of which package to use?